### PR TITLE
reduce multisig2 contract size by replacing panic code with env::panic_str(...)

### PR DESCRIPTION
**Pull request by adsick**
Addresses #184
the gain in size is not huge but the changes made multisig2.wasm 735 bytes lighter.

### CODE REVIEW

File: multisig2/src/lib.rs [hunk 1]
```diff
@@ -449,17 +457,17 @@ mod tests {
...
-    use near_sdk::Balance;
 
     use super::*;
 
     /// Used for asserts_eq.
     /// TODO: replace with derive when https://github.com/near/near-sdk-rs/issues/165
     impl Debug for MultiSigRequest {
         fn fmt(&self, _f: &mut Formatter<'_>) -> Result<(), Error> {
-            panic!("Should not trigger");
+            env::panic_str("Should not trigger");
```
Comment (ilblackdragon) [inline multisig2/src/lib.rs:470]: you don't need to update tests, as they are not contributing to the size of the contract.

File: multisig2/src/lib.rs [hunk 2]
```diff
@@ -127,11 +128,12 @@ impl MultiSigContract {
     /// @params num_confirmations: k of n signatures required to perform operations.
     #[init]
     pub fn new(members: Vec<MultisigMember>, num_confirmations: u32) -> Self {
-        assert!(!env::state_exists(), "Already initialized");
-        assert!(
```
Comment (ilblackdragon) [inline multisig2/src/lib.rs:131]: it's easier to replace this with to keep the same number of instructions but keep the code pretty much the same.
```
#[inline]
fn assert(condition: bool, error: &str) {
    if condition {
        env::panic_str(error);
    }
}
```

**Review by ilblackdragon — COMMENTED** [review]
Will merge this in but leaving comments for future.

File: multisig2/src/lib.rs (hunk 1 above)
Comment (adsick) [reply multisig2/src/lib.rs:470]: ah, yea, was thinking about that too but decided to go all in)

**Comment by ilblackdragon** [comment]
It's merged directly into #125

**Pull request closed without merging**